pkg/metadata/alias.go

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2022 Zinc Labs Inc. and Contributors
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package metadata
+
+import (
+	"github.com/goccy/go-json"
+)
+
+type alias struct{}
+
+var Alias = new(alias)
+
+func (t *alias) Set(data map[string][]string) error {
+	buf, err := json.Marshal(data)
+	if err != nil {
+		return err
+	}
+	return db.Set("/aliases/alias", buf)
+}
```

Comment:
Alias is a top level concept, Can we set alias by alias, not all alias as an entry. If we have over hundred alias, maybe set alias isolate will better.

Reply:
For now, we just try to keep the entire alias map in sync with metadata storage.

Reply:
Okay, i suggest split it in the feature.